CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - added: Paybis sell to debit card
 - changed: Change FlipInput styling to make the edit functionality more obvious
 - changed: Move asset-specific settings into their own settings page
+- fixed: Experiment config generation for more than 2 variants
```

Comment:
`fixed: Write updated experiment configs to disk` and `changed: Experiment config probability distribution support percentage based values`